docs/tutorial/t-deploy.md

```diff
@@ -0,0 +1,32 @@
+# Make Self Signed Certificates Operator up and running
```

Comment:
As mentioned in other places, what is here could be in the index. We probably don't want a full page to tell people how to deploy charms.